server/stream.go

```diff
@@ -6028,16 +6028,17 @@ func (mset *stream) clearPreAck(o *consumer, seq uint64) {
 }
 
 // ackMsg is called into from a consumer when we have a WorkQueue or Interest Retention Policy.
-func (mset *stream) ackMsg(o *consumer, seq uint64) {
+// Returns whether the message at seq should be removed as a result of the ACK.
```

Comment:
I think this would be better if it was `didRemove` sine it will be removed in this function.

Reply:
Done